packages/react-ui-core/src/Ratings/LinearGradient.js

```diff
@@ -15,17 +16,34 @@ export default class LinearGradient extends Component {
   render() {
     const {
       id,
-      color,
+      fillColor,
+      backgroundFillColor,
       width,
     } = this.props
 
+    let backgroundStopOpacity = 1
+    let effectiveBackgroundFillColor = backgroundFillColor
+
+    if (!effectiveBackgroundFillColor) {
+      effectiveBackgroundFillColor = fillColor
```

Comment:
Does it matter if this is set if opacity is 0 still?

Reply:
I'm not really sure, the last two stops were setting color to fillColor before, so i wanted to render the same markup as it was doing previously if no backgroundFillColor was provided.